src/index.html

```diff
@@ -17,6 +17,75 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="./index.html"
+        class="headerimg"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo"
+        />
+      </a>
+
+      <nav class="navlist">
+        <a
+          href="./index.html"
```

Comment:
```suggestion
          href="#apple"
```
Change all other links